assets/queries/k8s/tiller_deployment_is_accessible_within_cluster/query.rego

```diff
@@ -70,18 +73,19 @@ CxPolicy[result] {
 
 	isTillerTemplate(document)
```

Comment:
the method checkmetadata() is used in the query 'tiller_is_deployed' move it to the libraries and import it to prevent code duplication